src/calendar-list/index.js

```diff
@@ -59,7 +59,9 @@ class CalendarList extends Component {
     /** How far from the end to trigger the onEndReached callback */
     onEndReachedThreshold: PropTypes.number,
     /** Called once when the scroll position gets within onEndReachedThreshold */
-    onEndReached: PropTypes.func
+    onEndReached: PropTypes.func,
+    /** Allows FlatList to be displayed on scrollView */
```

Comment:
That's a ScrollView prop (not FlatList's) and its description reads "Enables nested scrolling for Android API level 21+". Please mantion it.